data/augs.py

```diff
@@ -3,33 +3,36 @@
 
 """
 
-#from models.backtranslation import BackTranslationModel
 import torch
 import numpy as np
-
-
-class Compose:
-    """
-    Compose several augmentations together. Based on torchvision's `Compose`
+from typing import Dict
+import sys
+import json
+import multiprocess
+# specifies a dictionary of architectures
+_AUGS: Dict[str, any] = {}  # registry
+
+
+def register_aug(name):
+    """Decorator used register a CARP architecture

```

Comment:
I think this docstring is incorrect... you are registering the augmentations, not the CARP architecture, right?